docs/atomic-cluster.1.md

```diff
@@ -0,0 +1,65 @@
+% ATOMIC(1) Atomic Man Pages
+% Matthew Barnes
+% April 2016
+# NAME
+atomic-cluster - Manage cluster hosts
+
+# SYNOPSIS
+**atomic cluster [OPTIONS] SUBCOMMAND**
+
+**atomic cluster** will issue commands to a Commissaire server over HTTP.
+
+Commissaire allows administrators of a Kubernetes, Atomic Enterprise or
+OpenShift installation to perform administrative tasks without the need
+to write custom scripts or manually intervene on systems.
+
+# NOTE
+The `cluster` subcommand is only available when the Commissaire client
+module `commctl` is installed.
+
+# OPTIONS
+**-h** **-help**
+  Print usage statement.
+
+# SUBCOMMANDS
+**get cluster** CLUSTER_NAME
+  Report status of the specified cluster.
```

Comment:
There are other options to ```commctl```'s```cluster``` command (```get```, ```create```, ```delete```). However, seeing ```cluster get cluster $NAME``` does seem odd.

A few ideas:

* Maybe it would make better sense to use a different direct command than ```cluster```. Maybe ```manage```?
* Maybe we can flatten out the commands we add so that they are at a higher level like:

```
atomic cluster get $NAME
atomic host delete $NAME
atomic status # maybe this one is a special case where we say atomic manager status
...
```